### PR TITLE
(Bug) #168 Fix printing of log context to console

### PR DESCRIPTION
**Pull request by yaroslav-fedyshyn-nordwhale**
# Description

Fix the way of getting log context while formatting print string.

About #168 

# How Has This Been Tested?

Check the printed log information in server console.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/imports/logger.js [hunk 1]
```diff
@@ -52,8 +56,8 @@ const logger = winston.createLogger({
     timestamp(),
     format.errors({ stack: true }),
     printf(({ level, timestamp, from, userId, ...rest }) => {
-      const logPayload = mapKeys(rest, (_, key) => (key === SPLAT ? 'context' : key))
-      const stringifiedPayload = JSON.stringify(logPayload, (_, logValue) => formatLogValue(logValue))
+      const context = rest[SPLAT]
```
Comment (serdiukov-o-nordwhale) [inline src/imports/logger.js:59]: so, [SPLAT] proparty should be in the payload, we have to **copy** it to context not just rename ?

**Comment by serdiukov-o-nordwhale** [comment]
@yaroslav-fedyshyn-nordwhale as we have to serialize error (instead of format as the sitring), i've used pino's serialized 
you already know it from the redact task - it converts errors to { type, stack, message } objects.
please check if it will be enough to satisfy task conditions

**Comment by sirpy** [comment]
@yaroslav-fedyshyn-nordwhale this still happens duplicate data in the "context" variable.

12 Aug 2020 14:42:17.149861 <190>1 2020-08-12T11:42:16.797194+00:00 app web.1 - - [32m2020-08-12T11:42:16.796Z - workerId:0 - debug (FROM /verify/face/440c215A01f47F4180A18EC49F52b5F7c23EEc84 ): {"uuid":"Pkd4+ABii6","data":{"glasses":false,"sessionTokenStatus":1,"faceMapType":0,"livenessStatus":1,"isLowQuality":false},"meta":{"code":400,"message":"3D FaceMaps that are used with Search APIs must have had Liveness Proven.","ok":false,"m":{"prepareAndUpdateTimeInMS":2806,"totalDurationInMS":2867}},"message":"Received response from Zoom API:","context":[{"data":{"glasses":false,"sessionTokenStatus":1,"faceMapType":0,"livenessStatus":1,"isLowQuality":false},"meta":{"code":400,"message":"3D FaceMaps that are used with Search APIs must have had Liveness Proven.","ok":false,"m":{"prepareAndUpdateTimeInMS":2806,"totalDurationInMS":2867}}}]}[39m

**Comment by yaroslav-fedyshyn-nordwhale** [comment]
> Received response from Zoom API

@sirpy I recheck the code and found what cause duplicate data when logging. I pushed a fix to the master - see commit - 3c1044eaa18a3711fd35efa2e908de15bc2cd129

**Pull request merged**